build.sh

```diff
@@ -218,7 +218,7 @@ for i in "$@"
         ;;
         arm64)
         __BuildArch=arm64
-        __MSBuildBuildArch=arm64
+        __MSBuildBuildArch=AnyCPU
```

Comment:
In error, sorry, I was looking through and couldn't see any definition for arm64 in MSBuild.